docs/reference-implementation/APIM.md

```diff
@@ -0,0 +1,141 @@
+# Integrating APIM with AlwaysOn-Connected 
+The following guide will help you integrate APIM with the current AlwaysOn-Connected framework to route traffic to your AKS cluster from Front Door. The implementation is broken into three stages: [establishing Front Door to APIM connectivity](#establishing-front-door-to-apim-connectivity), [privatizing the AKS cluster](#privatize-aks-cluster), and [implementing dynamic generation of api definitions in the deployment pipeline](#implement-dynamic-generation-of-api-definitions-in-deployment-pipeline).
+
+## Establishing Front Door to APIM Connectivity
+See [this branch](https://github.com/Azure/Mission-Critical-Connected/tree/feature/apim) for implementation or follow the steps below.
+
+1. Go to the [stamp resources folder](/src/infra/workload/releaseunit/modules/stamp/) and add a file named "api_management.tf" with the following contents:
```

Comment:
since you added the other files also as example files, I would rather do this for apim.tf as well, instead of pasting the code here